content/en/quickstart/create-device.md

```diff
@@ -11,3 +11,105 @@ menu:
     weight: 10
 toc: true
 ---
+This guide will cover how to get the first steps while using the plgd cloud in the Raspberry Pi. This includes how to create cloud server (OCF Device) available from IoTivity-Lite open source project on the Github.
+
+## Requirements
+You will need the following: 
+- Raspberry Pi board with a microSD card with [Raspberry Pi OS](https://www.raspberrypi.org/software/operating-systems/#raspberry-pi-os-32-bit)
+- Optional: HDMI monitor and cable, USB keyboard and mouse for the Pi
+- Ethernet or Wi-Fi connection between a DHCP-enabled network and the Pi
+- Personal computer(Linux/MacOS recommended) on the same network as the Pi
+
+## Setting up the Raspberry Pi
+1. Set up your SD card as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/2)
```

Comment:
This guide is very nice and we don't need to list here all their steps in the bullet list. Just write:
To set up your Raspberry Pi please follow [this step-by-step tutorial](...).